src/util/json_output.ts

```diff
@@ -232,8 +233,10 @@ function generatePropertyMap(
         const unit = contract.vScope;
         const predRange = annotation.predicateFileLoc;
         const annotationRange = annotation.annotationFileRange;
-        const debugEvent = ctx.debugEventDefs.get(annotation.id);
-        const signature = debugEvent !== undefined ? debugEvent.canonicalSignature : "";
+        const signature = ctx.debugEventsSignature;
```

Comment:
Lets remove the `signature` variable and the `debugEventSignature: signature` field from the struct on line 304. This is no longer neccessary as the debug event signature is always the same